package.json

```diff
@@ -19,13 +19,14 @@
     "grpc-web": "1.0.7"
   },
   "scripts": {
-    "build": "tsc -d",
-    "clean": "rm -rf ./generated ./dist ./build",
-    "pretest": "./scripts/regenerate_protos.sh && tsc",
-    "lint": "eslint . --ext .ts --fix",
-    "test": "npm run lint && nyc mocha",
-    "prepublishOnly": "npm test && tsc -d",
-    "webpack": "npm run clean && ./scripts/regenerate_protos.sh && tsc && npx webpack"
+    "build": "rm -rf ./src/generated ./build && ./scripts/regenerate_protos.sh && npm run lint && tsc -d",
```

Comment:
Can you update this for @hbergren's build changes? Specifically I think he removed `lint:fix`.

Also think that you should just call `npm run clean` as the first step of thebuild command.

Reply:
Fixed, I do think we should have kept fix and lint as two separate commands though.

Reply:
Why do you prefer separate steps for linting and fixing?

My thinking is that anything that can be automatically fixed just should be, without you thinking about it. `lint` is the command usually run in CI, but I don't feel like CI should fail because someone forgot to run Prettier before pushing.